test-suite/buildtests/layer-02-maximus.opts

```diff
@@ -111,7 +111,7 @@ DISTCHECK_CONFIGURE_FLAGS=" \
 	--with-pthreads \
 	--enable-build-info=squid\ test\ build \
 	--enable-ssl-crtd \
-	--with-openssl \
+	--with-openssl=${with_openssl:-yes} \
```

Comment:
> Compiling on Mac with homebrew requires passing through a custom location of openssl libraries

I am hearing two things in the above statement:

1. Enabling Squid feature F may require finding library FL in a custom location.
2. test-builds.sh needs a new mechanism to support custom build options, such as a custom FL location.

Items 1 is a fact. Item 2 is correct if either there is no existing usable mechanism or we find all existing usable mechanisms inferior to the proposed change. This brings up two questions:

1. Why our ./configure is incapable of finding OpenSSL in a custom location in a homebrew testing environment? No valid pkg-config? Missing custom pkg-config path? No pkg-config support on that platform?
2. If pkg-config cannot help, then why not use LIBOPENSSL_LIBS environment variable that seems to be designed for solving similar customization problems where pkg-config fails?